# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

moveBehaviorActionGoal action_goal
moveBehaviorActionResult action_result
moveBehaviorActionFeedback action_feedback
